packages/design-system/src/styles/components/_SingleInputDateField.scss

```diff
@@ -8,6 +8,7 @@
 // the source rules.
 // @import 'react-day-picker/dist/style.css';
 
+/* stylelint-disable declaration-no-important, no-duplicate-selectors -- Ignore inlined 3rd party styles */
```

Comment:
As explained in the comment above, we're inlining 3rd party CSS to guarantee it loads in the correct order. Ignoring a few linter rules for the 3rd party styles only.